server/xpub-server/entities/manuscript/resolvers/submitManuscript.test.js

```diff
@@ -103,8 +106,9 @@ describe('Manuscripts', () => {
         actualContent,
         actualIp,
       ] = mecaExport.mock.calls[0]
+
       expect(actualManuscript.id).toBe(id)
-      expect(actualContent).toBe('A real PDF')
+      expect(actualContent).toHaveBeenCalled()
```

Comment:
No point having this assertion, as we're testing the mock.

`expect(mecaExport).toHaveBeenCalled()` is enough.

Reply:
We also want to test that there has been an attempt to retrieve the content

Reply:
... this is now done elsewhere so removed.